container_pipeline/workers/test.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+
+import hashlib
+import json
+import logging
+import os
+
+from container_pipeline.utils import Build, get_job_name
+from container_pipeline.lib.log import load_logger
+from container_pipeline.lib.openshift import Openshift, OpenshiftError
+from container_pipeline.workers.base import BaseWorker
+
+
+class TestWorker(BaseWorker):
+    """Test Worker"""
+
+    def __init__(self, logger=None, sub=None, pub=None):
+        super(TestWorker, self).__init__(logger=logger, sub=sub, pub=pub)
+        self.openshift = Openshift(logger=self.logger)
+
+    def test(self, job):
+        """Runs openshift test job"""
+        namespace = get_job_name(job)
+        project = hashlib.sha224(namespace).hexdigest()
+
+        try:
+            self.openshift.login()
+            build_id = self.openshift.build(project, 'test')
+            if not build_id:
+                return False
+        except OpenshiftError as e:
+            logger.error(e)
+            return False
+
+    def handle_test_success(self, job):
+        """Handle build success for job"""
+        self.logger.debug("Test is successful going for next job")
+
+    def handle_test_failure(self, job):
+        """Handle build failure for job"""
```

Comment:
Looking at the two commits in this PR, I guess you suggested to add newline after the docstring. But [PEP 257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) recommends otherwise for one-line docstrings.

> There's no blank line either before or after the docstring.

@navidshaikh @mohammedzee1000